src/class.js

```diff
@@ -0,0 +1,53 @@
+export function Task(description, index) {
+  this.description = description;
+  this.completed = false;
+  this.index = index;
+}
+
+export class TaskList {
+  constructor() {
+    this.container = [];
+    this.index = 0;
+  }
+
+  add(title) {
+    this.container = JSON.parse(window.localStorage.getItem('tasklist'));
+    this.index = JSON.parse(window.localStorage.getItem('tasknumber'));
+    if (this.container === null) {
+      this.container = [];
+    }
+    if (this.index === null) {
+      this.index = 0;
+    }
+    const newTask = new Task(title, this.index);
+    this.container.push(newTask);
+    this.index += 1;
+    console.log(this.index);
+    console.log(this.container);
```

Comment:
- [ ] **Required:** Your application data is available in the console. It is unprofessional to do this. Ensure you remove all `console.log()`s before committing them.